torch/csrc/jit/codegen/cuda/test/test_gpu.cpp

```diff
@@ -23574,6 +23574,77 @@ TEST_F(NVFuserTest, FusionReproNoncontigBroadcast_CUDA) {
       executor_cache.fusion(), cg_outputs, {t0, t1}, {t2}, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, TestTransformPropagatorSibling_CUDA) {
```

Comment:
In our convention, the test name should starts with `Fusion`, so something like `FusionTransformPropagateSibling_CUDA`. Not sure if it needs to be that way.

Reply:
fixed